sites/next.skeleton.dev/src/content/docs/components/accordion/react.mdx

```diff
@@ -10,7 +10,7 @@ import { Page as ExampleCollapsible } from '@examples/components/accordions/Exam
 import ExampleCollapsibleRaw from '@examples/components/accordions/ExampleCollapsible.tsx?raw';
 import { Page as ExampleMultiple } from '@examples/components/accordions/ExampleMultiple.tsx';
 import ExampleMultipleRaw from '@examples/components/accordions/ExampleMultiple.tsx?raw';
-import { AccordionSchema } from '@skeletonlabs/skeleton-react/schemas';
```

Comment:
It would be great to break out the schema import separately from component imports. Just for readability in the code:

![image](https://github.com/user-attachments/assets/0daaa6e1-ea13-43f0-9836-1618ffa492c2)

Reply:
Schemas are now gone as they are auto detected by the URL! (unless explicitely passed 😄)